src/ui/public/kuery/functions/is.js

```diff
@@ -17,17 +18,12 @@ export function buildNodeParams(fieldName, value, serializeStyle = 'operator') {
 }
 
 export function toElasticsearchQuery(node, indexPattern) {
-  const { arguments: [ fieldNameArg, valueArg ] } = node;
+  const { arguments: [fieldNameArg, valueArg] } = node;
   const fieldName = literal.toElasticsearchQuery(fieldNameArg);
-  const field = indexPattern.fields.byName[fieldName];
   const value = !_.isUndefined(valueArg) ? literal.toElasticsearchQuery(valueArg) : valueArg;
 
-  if (field && field.scripted) {
-    return {
-      script: {
-        ...getPhraseScript(field, value)
-      }
-    };
+  if (fieldName === '*' && value === '*') {
+    return { match_all: {} };
   }
   else if (fieldName === null) {
```

Comment:
When the user types in a raw value without a field, we need to send that query without any field specified so the default field in the index settings will take effect. This gives users a way to avoid the ES query max field limit, which we first noticed with the [beats dashboards](https://github.com/elastic/beats/issues/5275).